kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -20,14 +22,17 @@ public OddNumbersExterminator() {
         return evenList;
     }
 
-   public int checkList(ArrayList <Integer> tempNumbers) {
+    @VisibleForTesting
+   public boolean isEven(ArrayList <Integer> tempNumbers) {
 
-        int checksum = 0;
+        boolean evenValidator = true;
 
         for(int tempNumber : tempNumbers ) {
-            checksum += tempNumber;
+            if (tempNumber % 2 != 0) {
+            evenValidator = false;
```

Comment:
jeśli znajdzie jakąkolwiek nieparzystą, to wiemy już, że powinien zwrócić false, a więc usuń zmienną pomoczniczą. 
`for (int n : tempNumbers) {
  if (n % 2 != 0) {
    return false;
  }
  return true;
}`